python/catkin/builder.py

```diff
@@ -972,7 +972,7 @@ def build_workspace_isolated(
             sys.exit('Packages not found in the workspace: %s' % ', '.join(unknown_packages))
 
     # Report topological ordering
-    ordered_packages = topological_order_packages(packages, None, blacklisted)
+    ordered_packages = topological_order_packages(packages, whitelisted=None, blacklisted=ignore_packages)
```

Comment:
As in my [above](https://github.com/ros/catkin/pull/1027#discussion_r314084468) example there is no need to pass `whitelisted=None`. Please remove it from the list of arguments.

Reply:
Oups, done.